httpbin/handlers_test.go

```diff
@@ -1989,21 +1989,17 @@ func TestStreamBytes(t *testing.T) {
 
 		// as is negative chunk size
 		{"/stream-bytes/256?chunk_size=-10", 256},
+
+		// way too large chunk size is okay
+		{fmt.Sprintf("/stream-bytes/256?chunk_size=%d", maxBodySize+20), 256},
 	}
 	for _, test := range okTests {
 		t.Run("ok"+test.url, func(t *testing.T) {
 			r, _ := http.NewRequest("GET", test.url, nil)
 			w := httptest.NewRecorder()
 			handler.ServeHTTP(w, r)
 
-			// TODO: The stdlib seems to automagically unchunk these responses
-			// and I'm not quite sure how to test this:
-			//
-			//     assertHeader(t, w, "Transfer-Encoding", "chunked")
-			//
-			// Instead, we assert that we got no Content-Length header, which
-			// is an indication that the Go stdlib streamed the response.
-			assertHeader(t, w, "Content-Length", "")
```

Comment:
As mentioned above, I'd rather maintain this (admittedly odd) test, or figure out a way to actually test the streaming behavior.